Sources/TuistAutomation/XcodeBuild/XcodeBuildController.swift

```diff
@@ -79,7 +74,7 @@ public final class XcodeBuildController: XcodeBuildControlling {
             command.append(contentsOf: ["-derivedDataPath", derivedDataPath.pathString])
         }
 
-        return try run(command: command, isVerbose: environment.isVerbose)
+        return try run(command: command, rawXcodebuildLogs: rawXcodebuildLogs)
```

Comment:
Should we still default to xcodebuild logs if `--verbose` is passed?

Reply:
Yeah, I think that's fine. I'll add it